e2e/test/scenarios/embedding/embedding-linked-filters.cy.spec.js

```diff
@@ -105,7 +105,7 @@ describe("scenarios > embedding > dashboard > linked filters (metabase#13639, me
 
       popover().within(() => {
         testPairedTooltipValues("STATE", "AK");
-        testPairedTooltipValues("Count", "1");
+        testPairedTooltipValues("count", "1");
```

Comment:
We should not change specs like this. If this fails, it means we broke the behavior and should fix it